workflow/log-monitoring/custom-metrics.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Custom Metrics
+sidebarTitle: Custom Metrics
+---
+
+The Custom Metrics section allows you to create filters that graphically display your log data. This feature is useful for transforming your raw data into a digestible format and drilling directly into the Log Explorer
```

Comment:
"The Custom Metrics section allows you to create graphically representations of your log data. This feature is useful for transforming your raw data into a visual format, or for aggregating across and drilling deeper into oyur logs." (also there was a missing period at the end)